Editor/CoreAPI/CoreApi.cs

```diff
@@ -34,7 +35,7 @@ public class CoreApi
 
         public CoreApi()
         {
-            _settingsStore = new SettingsStore(_fileWrapper, settingsFilePath: Paths.PluginSettingsFile);
+            _settingsStore = new Settings<SettingsKeys>(Paths.PluginSettingsFile);
```

Comment:
Thanks for making this update! I like that it's consolidated now. I think `GameLiftServer.cs` is the only entity creating it's own Settings object. Can we flip that over to use `CoreApi` too? That can be a different PR if needed.

Reply:
As thats in the SampleGame assembly intended to be built and run on the server, no Editor scripts can be included. This change was done to `EditorCoreApi`, thus cannot be included in the Server. The change to `Settings<T>` stemmed from this issue, and it is in the `Runtime` assembly